docs/docs/en/guide/resource/configuration.md

```diff
@@ -2,13 +2,23 @@
 
 The Resource Center is usually used for operations such as uploading files, UDF functions, and task group management. You can appoint the local file directory as the upload directory for a single machine (this operation does not need to deploy Hadoop). Or you can also upload to a Hadoop or MinIO cluster, at this time, you need to have Hadoop (2.6+) or MinIO or other related environments.
 
+## Local File Resource Configuration
+For a single machine, you can choose to use local file directory as the upload directory (no need to deploy Hadoop) by making the following configuration.
+
+### Configuring the common.properties
```

Comment:
If we using code or configuration file in the docs, should better using the code block in markdown syntax
```suggestion
### Configuring the `common.properties`
```